### ISSUE TITLE
unable to build librust-libnotcurses-sys 2.1.7 deb

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1319]
Attempting to build and submit `librust-libnotcurses-sys` 2.1.7 for Debian fails with the following output:

```
[libnotcurses-sys 2.1.7] thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Failure { command: "\"x86_64-linux-gnu-pkg-config\" \"--libs\" \"--cflags\" \"notcurses\" \"notcurses >= 2.1.7\"", output: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "Package tinfo was not found in the pkg-config search path.\nPerhaps you should add the directory containing `tinfo.pc\'\nto the PKG_CONFIG_PATH environment variable\nPackage \'tinfo\', required by \'notcurses-core\', not found\n" } }', build/build.rs:12:10
[libnotcurses-sys 2.1.7] stack backtrace:
[libnotcurses-sys 2.1.7]    0: rust_begin_unwind
[libnotcurses-sys 2.1.7]              at /usr/src/rustc-1.48.0/library/std/src/panicking.rs:483
[libnotcurses-sys 2.1.7]    1: core::panicking::panic_fmt
[libnotcurses-sys 2.1.7]              at /usr/src/rustc-1.48.0/library/core/src/panicking.rs:85
[libnotcurses-sys 2.1.7]    2: core::result::unwrap_failed
[libnotcurses-sys 2.1.7]              at /usr/src/rustc-1.48.0/library/core/src/result.rs:1220
[libnotcurses-sys 2.1.7]    3: core::result::Result<T,E>::unwrap
[libnotcurses-sys 2.1.7]              at /usr/src/rustc-1.48.0/library/core/src/result.rs:973
[libnotcurses-sys 2.1.7]    4: build_script_build::main
[libnotcurses-sys 2.1.7]              at ./build/build.rs:9
[libnotcurses-sys 2.1.7]    5: core::ops::function::FnOnce::call_once
[libnotcurses-sys 2.1.7]              at /usr/src/rustc-1.48.0/library/core/src/ops/function.rs:227
[libnotcurses-sys 2.1.7] note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
error: failed to run custom build command for `libnotcurses-sys v2.1.7 (/<<PKGBUILDDIR>>)`

Caused by:
  process didn't exit successfully: `/<<PKGBUILDDIR>>/target/debug/build/libnotcurses-sys-e1ac9e2f634dde57/build-script-build` (exit code: 101)
```

I can get `tinfo` via `pkg-config` just fine in my external environment:

```
[schwarzgerat](1) $ pkg-config --libs tinfo
-ltinfo
[schwarzgerat](0) $ 
```

so there is presumably a missing dep in the chroot.

@joseluis , i've got this, i think.

**Comment by dankamongmen** [comment]
Hrmmm, I'm thinking that maybe `libnotcurses-dev` needs a dep on `libncurses-dev`?

```
Depends: libnotcurses2 (= 2.1.7+dfsg.1-2), libnotcurses-core-dev (>= 2.1.7+dfsg.1-2)
```

**Comment by dankamongmen** [comment]
nah, `libnotcurses-dev` oughtn't depend on `libncurses-dev`; it doesn't publicly import any of the relevant headers. i've added a dep to the packaging for `librust-libnotcurses-sys`. Let's see if that fixes it. In the meantime, do we need the explicit tinfo dep in the rust code?

**Comment by dankamongmen** [comment]
> nah, `libnotcurses-dev` oughtn't depend on `libncurses-dev`; it doesn't publicly import any of the relevant headers. i've added a dep to the packaging for `librust-libnotcurses-sys`. Let's see if that fixes it. In the meantime, do we need the explicit tinfo dep in the rust code?

there is no such explicit dep.

**Comment by dankamongmen** [comment]
Yeah, adding that dep appears to have resolved things. Let's get this uploaded to the archive. See #1290.

**Comment by dankamongmen** [comment]
https://salsa.debian.org/rust-team/debcargo-conf/-/merge_requests/179. I think we can mark this closed, and track the upload in #1290.
